tests/sps/validation/test_alternatives.py

```diff
@@ -129,64 +130,14 @@ def test_validation_parent_fail(self):
-                        <body>
-                            <disp-formula>
-                                <alternatives>
-                                    <mml:math />
-                                    <tex-math />
-                                </alternatives>
-                            </disp-formula>
-                        </body>
-                    </sub-article>
                 </article>
                 """
         )
         params = {
             "table-wrap": ["graphic", "table"],
             "fig": ["graphic", "media"]
         }
-        obtained = list(AlternativesValidation(self.xmltree, params).validation())
-        expected = [
-            {
-                'title': 'Alternatives validation',
-                'parent': 'article',
-                'parent_id': None,
-                'item': 'inline-formula',
-                'sub_item': 'alternatives',
-                'validation_type': 'value in list',
-                'expected_value': ['table-wrap', 'fig'],
-                'got_value': 'inline-formula',
-                'response': 'ERROR',
-                'message': "Got inline-formula, expected ['table-wrap', 'fig']",
-                'advice': "Provide parent tag according to the list: ['table-wrap', 'fig']",
-                'data': {
-                    'alternative_children': ['{http://www.w3.org/1998/Math/MathML}math', 'tex-math'],
-                    'alternative_parent': 'inline-formula',
-                    'parent': 'article',
-                    'parent_id': None
-                }
-            },
-            {
-                'title': 'Alternatives validation',
-                'parent': 'article',
-                'parent_id': None,
-                'item': 'inline-formula',
-                'sub_item': 'alternatives',
-                'validation_type': 'value in list',
-                'expected_value': None,
-                'got_value': ['{http://www.w3.org/1998/Math/MathML}math', 'tex-math'],
-                'response': 'ERROR',
-                'message': "Got ['{http://www.w3.org/1998/Math/MathML}math', 'tex-math'], expected None",
-                'advice': 'Provide child tags according to the list: None',
-                'data': {
-                    'alternative_children': ['{http://www.w3.org/1998/Math/MathML}math', 'tex-math'],
-                    'alternative_parent': 'inline-formula',
-                    'parent': 'article',
-                    'parent_id': None
-                }
-            }
-        ]
-        for i, item in enumerate(expected):
-            with self.subTest(i):
-                self.assertDictEqual(item, obtained[i])
\ No newline at end of file
+        obtained = AlternativesValidation(self.xmltree, params)
+        with self.assertRaises(ValidationAlternativesException) as context:
+            next(obtained.validation())
+        self.assertEqual("Tag inline-formula is not provided in the function parameters", str(context.exception))
```

Comment:
@Rossi-Luciano no lugar de "Tag inline-formula is not provided in the function parameters", use "inline-formula is not configured to use 'alternatives'. Provide alternatives parent and children"

Reply:
OK, @robertatakenaka